uint8 id_module
uint8 submode
uint8 status